### PR TITLE
fix: update favorite-markets-config

### PR DESCRIPTION
**Pull request by jaredvu**
- add helper function

### CODE REVIEW

File: src/state/migrations/3.ts [hunk 1]
```diff
@@ -1,13 +1,30 @@
-import { PersistedState } from 'redux-persist';
+import { PersistedState, PersistState } from 'redux-persist';
 
-import { AppUIConfigsState, initialState as appUiConfigsInitialState } from '../appUiConfigs';
+import { DisplayUnit } from '@/constants/trade';
```
[the remark this answers is not in the document]
Comment (jaredvu) [reply src/state/migrations/3.ts]: added DisplayUnit

File: src/state/migrations/3.ts [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+
+import {
+  AppColorMode,
+  AppThemeSetting,
+  initialState as appUiConfigsInitialState,
+  AppUIConfigsState,
+} from '../appUiConfigs';
```
[the remark this answers is not in the document]
Comment (jaredvu) [reply src/state/migrations/3.ts]: added types inline

File: src/state/migrations/__test__/3.test.ts [hunk 3]
```diff
@@ -2,17 +2,14 @@ import { afterEach, describe, expect, it } from 'vitest';
 
 import { DisplayUnit } from '@/constants/trade';
 
-import {
-  AppColorMode,
-  AppTheme,
-  initialState as appUiConfigsInitialState,
-} from '@/state/appUiConfigs';
+import { AppColorMode, AppTheme } from '@/state/appUiConfigs';
```
Comment (tyleroooo) [inline src/state/migrations/__test__/3.test.ts:5]: gotta kill these too sadly

**Review by tyleroooo — COMMENTED** [review]
other than that, we good

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**